### ISSUE TITLE
Remove spin box arrow from input of type number

### ISSUE THREAD

**Issue by slim-hmidi** [necolas/normalize.css#752]
I don't know if it is the suitable place to mention this feature but I actually using `normalize` in my project and I'm using a [TextField](https://material-ui.com/demos/text-fields/) of type `number`. The component contains the spin arrows box which are annoying and it looks like [TextField Example](https://codesandbox.io/s/1r77zz0m7j). 
According to this reference [::-webkit-inner-spin-button](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-inner-spin-button) and [CSS: Hide “spinners” (up/down arrows) on inputs with type=”number” using a class](https://www.tilcode.com/css-hide-spinners-updown-arrows-on-inputs-with-typenumber-using-a-class/) I added a `style.css` to hide the spin arrow box. This is the reference [TextField without spin arrow box](https://codesandbox.io/s/n53n09147m).
Is not possible to add this feature in `normalize.css` or there are other constraints?

**Comment by mvasilkov** [comment]
It's a useful accessibility feature. IMO normalize.css shouldn't disable useful accessibility features.

**Comment by brianWreaves** [comment]
> It's a useful accessibility feature. IMO normalize.css shouldn't disable useful accessibility features.
 
I agree accessibility shouldn't be disabled.  With that said, incorporating ARIA tags is of more value than those little-bitty buttons which are both an accessibility and usability issue.  The ability to use the arrow keys as well as typing the value is more practical for all users.  That's what should not be removed.
 
"The increase and decrease buttons are not keyboard focusable, but can be operated with a mouse." so there should be no accessibility issues (REF:  https://www.accede-web.com/en/guidelines/rich-interface-components/spinbuttons-simulated-using-aria/).  
 
Perhaps normalize could consider including CSS to increase the size of each spinner's clickable target area of the up/down arrows to at least half the full height of the field, if not remove them.

**Comment by Yeswanth-JG** [comment]
> "The increase and decrease buttons are not keyboard focusable...

But, Except IE11 and less, all other desktop browsers allow up & down arrow buttons to step the numbers.

**Comment by gurnoor-garry** [comment]
Hey 
I want to work on this issue. Could u please assign it to me?

**Comment by garrettw** [comment]
@gurnoor-garry Just submit a PR

**Comment by tijivarghese** [comment]
I agree with mvasilkov, the spinners or the arrows in the input type number is a good accessibility feature and should not be removed through normalize CSS. If it is not required then we can [hide the arrows](https://www.moreonfew.com/how-to-remove-or-hide-arrow-spinners-from-input-type-number/) or [style the arrows](https://stackoverflow.com/questions/26024771/styling-an-input-type-number) in our local CSS. Moreover, users are getting used to the arrows and it is a good feature too.
